ticdc/ticdc-open-api.md

```diff
@@ -0,0 +1,617 @@
+```
+
+If the request is successful, `202 Accepted` is returned. If the request fails, an error message and error code are returned.
+
+## Update the replication configuration
+
+This is an asynchronous interface. If the request is successful, `202 Accepted` is returned. The returned result only means that the server agrees to run the command but does not guarantee that the command will be run successfully.
+
+To modify the changefeed configuration, follow the steps of `pause the replication task -> modify the configuration -> resume the replication task`.
+
+### Request URI
+
+`PUT /api/v1/changefeeds/{changefeed_id}`
+
+### Parameter description
+
+#### Path parameters
+
+| Parameter name | Description |
+| :-------------- | :----------------------------------- |
+| `changefeed_id` | The ID of the replication task (changefeed) to be queried. |
+
+#### Parameters for the request body
+
+Currently, the following configuration can be modified only via the API.
+
+| Parameter name | Description |
+| :-------------------- | :-------------------------- --------------------------- |
+| `target_ts` | `UINT64` type. Specifies the target TSO of the changefeed. (Optional) |
+| `sink_uri` | `STRING` type. The downstream address of the replication task. (Optional) |
+| `filter_rules` | `STRING` type array. The rules for table schema filtering. (Optional) |
+| `ignore_txn_start_ts` | `UINT64` type array. Ignores the transaction of a specified start_ts. (Optional) |
+| `mounter_worker_num` | `INT` type. The mounter thread number. (Optional) |
+| `sink_config` | The configuration parameters of sink. (Optional) |
+
+The meanings of the above parameters are the same as those in the [Create a replication task](#create-a-replication-task) section. See that section for details.
+
+### Example
+
+The following request updates the `mounter_worker_num` of the replication task with the ID `test1` to `32`.
+
+{{< copyable "shell-regular" >}}
+
+```shell
+ curl -X POST -H "'Content-type':'application/json'" http://127.0.0.1:8300/api/v1/changefeeds/test1 -d '{"mounter_worker_num":32}'
+```
+
+If the request is successful, `202 Accepted` is returned. If the request fails, an error message and error code are returned.
+
+## Query the replication task list
+
+This is a synchronous interface. If the request is successful, the basic information of all nodes in the TiCDC cluster is returned.
+
+### Parameter description
+
+#### Query parameters
+
+| Parameter name | Description |
+| :------ | :---------------------------------------- ----- |
+| `state` | Optional. When this parameter is specified, the replication status information only of this state is returned. |
+
+The value options for `state` are `all`, `normal`, `stopped`, `error`, `failed`, and `finished`.
+
+If this parameter is not specified, the basic information of replication tasks in normal, stopped, and failed states is returned by default.
+
+### Request URI
```

Comment:
The `### Request URI` should be right after the secondary title `## Query the replication task list`

Reply:
done